src/zcash/zecPlugin.ts

```diff
@@ -72,7 +72,10 @@ export class ZcashTools implements EdgeCurrencyTools {
   }
 
   // will actually use MNEMONIC version of private key
-  async importPrivateKey(userInput: string): Promise<Object> {
+  async importPrivateKey(
+    userInput: string,
+    opts?: JsonObject
```

Comment:
I would do `opts: JsonObject = {}` to avoid the `?? {}` later. Things are easier to read when we get rid of `undefined` as early as possible. It also means that there is a single source of truth for what the default should be.